src/osd/OSD.cc

```diff
@@ -1051,6 +1051,13 @@ void OSDService::inc_osd_stat_repaired()
   return;
 }
 
+void OSDService::set_osd_stat_repaired(int64_t count)
+{
+  std::lock_guard l(stat_lock);
+  osd_stat.num_shards_repaired = count;
+  return;
```

Comment:
(sorry for missing this in the first round)
Please remove the 'return' line.
(and - if you can - please remove it from the inc_osd_stat_repaired() above, too.)

Reply:
Done!